src/custom/sdk/custom_instance.cc

```diff
@@ -100,6 +100,25 @@ CustomExecute(
   return instance->Execute(payload_cnt, payloads, input_fn, output_fn);
 }
 
+int
+CustomExecuteVer2(
+    void* custom_instance, const uint32_t payload_cnt, CustomPayload* payloads,
+    CustomGetNextInputVer2Fn_t input_fn, CustomGetOutputVer2Fn_t output_fn)
+{
+  if (custom_instance == nullptr) {
+    return ErrorCodes::Unknown;
+  }
+
+  CustomInstance* instance = static_cast<CustomInstance*>(custom_instance);
+  return instance->Execute(payload_cnt, payloads, input_fn, output_fn);
+}
+
+int
```

Comment:
Don't provide a default implementation, force the custom backend writer to think about which version they are implementing.